src/transcriptome.hpp

```diff
@@ -295,6 +295,9 @@ class Transcriptome {
         /// are ordered in reverse.
         void reorder_exons(Transcript * transcript) const;
 
+	/// Checks whether any adjacent exons are out of (strictly increasing) order
+	bool has_incorrect_order_exons(const vector<Exon> & exons) const;
+
         /// Checks whether any adjacent exons overlap.
         bool has_overlapping_exons(const vector<Exon> & exons) const;
```

Comment:
These both assume that the exons have already been through `reorder_exons()` and are now in increasing coordinate order, right? Maybe the comments here should specify that so that people know how to call them properly?